src/components/AboutJobsInfo.vue

```diff
@@ -3,189 +3,143 @@
         <div class="jobs-info-container">
             <img src="@/assets/images/online_research.svg" alt="" class="jobs-info-img">
             <div class="jobs-info-inner">
-                <h2 class="jobs-info-title">{{ $t ('about.jobsInfo.title') }}</h2>
+                <h2 class="jobs-info-title">{{ $t('about.jobsInfo.title') }}</h2>
                 <div class="jobs-info-subtitle">
-                    <p>{{ $t ('about.jobsInfo.subtitle') }}</p>
+                    <p>{{ $t('about.jobsInfo.subtitle') }}</p>
                 </div>
             </div>
             <div class="jobs-button-shadow">
-                <a href="#" class="jobs-btn">{{ $t ('about.jobsInfo.button') }}</a>
+                <a href="#" class="jobs-btn">{{ $t('about.jobsInfo.button') }}</a>
             </div>
         </div>
     </section>
 </template>
 <script>
 export default {
-    
+
 }
 </script>
-<style>
+
+<style lang="scss" scoped>
+@import "@/assets/styles/_variables.scss";
```

Comment:
https://stackoverflow.com/questions/53686780/setting-global-sass-variables-in-a-vue-project